### ISSUE TITLE
A11y Toolbar Being Inserted Before Skip Links

### ISSUE THREAD

**Issue by cgrymala** [UMWEDU/umw-outreach-pro-mods#11]
For accessibility reasons, the skip links should be the very first element on the page. However, the accessibility toolbar being used on the site is getting injected before the skip links, causing them to fall out of the proper DOM order.
